socorro/external/es/connection_context.py

```diff
@@ -141,37 +137,29 @@ def get_socorro_index_settings(self, mappings):
             'mappings': mappings,
         }
 
-    def create_socorro_index(self, index_name, mappings=None, log_result=False):
+    def create_index(self, index_name, mappings=None):
         """Create an index that will receive crash reports.
 
-        Note: This function can get called in two contexts: when the processor
-        is saving crash reports and also in the local dev environment scripts.
-        The former wants to ignore index-existing errors quietly but the latter
-        wants to log the result. Hence the fickle nature of this function.
+        :arg index_name: the name of the index to create
+        :arg mappings: dict of doctype->ES mapping
+
+        :returns: True if the index was created, False if it already
+            existed
 
         """
         if mappings is None:
             mappings = SuperSearchFields(context=self).get_mapping()
 
         es_settings = self.get_socorro_index_settings(mappings)
-        self.create_index(index_name, es_settings, log_result)
-
-    def create_index(self, index_name, es_settings, log_result=False):
```

Comment:
Way back in the day, we stored other things in ES and needed to be able to build indexes for things other than crash reports. We don't do that anymore, hence this refactoring to reduce the complexity.